pkg/server/share_manager.go

```diff
@@ -47,6 +60,11 @@ func NewShareManager(logger logrus.FieldLogger, volume volume.Volume) (*ShareMan
 	}
 	m.context, m.shutdown = context.WithCancel(context.Background())
 
+	err := m.makeLeaseClient()
```

Comment:
nit: suggest following the convention/pattern, e.g.
```
leaseClient, err := NewLeaseClient(ctx, volumeName)
if err != nil {
    ...
}

m.leaseHolder = ...
m.leaseHolder = ...
```

Reply:
Done, thanks.